charts/rancher-k3s-upgrader/0.3.2/Chart.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+name: rancher-k3s-upgrader
+description: Enables a k3s or rke2 cluster to update itself by reacting to Plan CRs.
+  Users do not need to manually upgrade this app. It will be automatically upgraded to the latest version when upgrading a cluster.
+home: https://github.com/rancher/system-charts/blob/dev-v2.6/charts/rancher-k3s-upgrader
+sources:
+  - "https://github.com/rancher/system-charts/blob/dev-v2.6/charts/rancher-k3s-upgrader"
+version: 0.3.2
+appVersion: v0.8.1
+kubeVersion: '>= 1.16.0-0'
```

Comment:
Should be <1.22.0-0 just like [the previous chart](https://github.com/rancher/system-charts/blob/6d474a45506c039f5a8559547a60df4e5ed0020c/charts/rancher-monitoring/v0.3.1/Chart.yaml#L12); the only changes we added inside were to modify the resource requirements, not support 1.22